src/securityPolicy.c

```diff
@@ -884,6 +957,7 @@ security_policy_t policyForSignTxMintConfirm(security_policy_t outputPolicy)
 		break;
```

Comment:
If I'm not mistaken, the `outputPolicy` variable here is actually mint policy from `policyForSignTxMintInit`, so we should
1. rename it,
2. remove the `POLICY_ALLOW_WITHOUT_PROMPT` case here, since it's unreachable,

right?

Reply:
I've renamed it, but I don't want to remove this currently unreachable switch case (it's OK to have this policy/UI machine ready, does not cost us a lot of code).